entrypoint.sh

```diff
@@ -38,6 +38,7 @@ fi
 # Setup Git config and push .pot file to github repo
 git config --global user.name "WordPress .pot File Generator"
 git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+git config --global --add safe.directory /github/workspace
```

Comment:
@odil-io fix looks good to me, but I think using `"$GITHUB_WORKSPACE"` here is a more future-proof solution. what do you think?

`git config --global --add safe.directory "$GITHUB_WORKSPACE"`

Reply:
Haven't tried that one yet, it should work right out of the box. I'll give it a go today

Reply:
Couldn't help myself and tested it right away. Works as expected!